packages/roblox-devtools-plugin/src/main.server.lua

```diff
@@ -0,0 +1,31 @@
+_G.__DEV__ = true
+-- _G.__DEBUG__ = true
+
+local RunService = game:GetService("RunService")
+
+local runModes = {
+	run = RunService:IsRunMode(),
+	studio = RunService:IsStudio(),
+	server = RunService:IsServer(),
+	client = RunService:IsClient(),
+	edit = RunService:IsEdit(),
+}
+
+if runModes.run then
+	return
+end
+
+if runModes.edit then
```

Comment:
Not good, I essentially make my entire game in Hoarcekat--I need it there!

Reply:
Yep! For now I want to disable it to prevent any confusion about the plugin not working in edit mode.

It's definitely possible to make it work though, the plugin would just need to make sure to re-connect and clean up the connections correctly.